src/libraries/System.Transactions.Local/src/System/Transactions/DtcProxyShim/NativeEnums.cs

```diff
@@ -115,7 +115,7 @@ internal enum OletxTransactionHeuristic : uint
     XACTHEURISTIC_DANGER = 4
 }
 
-internal enum OletxXactStat
+internal enum OletxXactStat : int
```

Comment:
Isn't int the default anyway? Is this about making this more explicit for some reason?

Reply:
Yes, this is the default. I added this as some of the COM definitions we’re using MarshalAs for this enum type and explicitly specifying UnmanagedType.I4, so I wanted to be consistent within the project (that specifies uint for cases where UnmanagedType.U4 was used).